scripts/demo-fruits.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+SCRIPTS_DIR="$(cd $(dirname "${BASH_SOURCE}") && pwd)"
+
+source ${SCRIPTS_DIR}/common.sh
+source ${SCRIPTS_DIR}/play-demo.sh
+
+####################################
+## Variables
+####################################
+VM_IP=${VM_IP:=127.0.0.1}
+NAMESPACE=app
+REGISTRY=localhost:5000
+IMAGE_VERSION=latest
+INGRESS_HOST=atomic-fruits.${VM_IP}.nip.io
+QUARKUS_APP_PATH="$HOME/atomic-fruits-service"
+APP_DIR=atomic-fruits
```

Comment:
As you do not use it, then it can be removed. It has been added as the project of Joze was part of a sub-project of thegit cloned repo